app/controllers/LongerCommodityCodeController.scala

```diff
@@ -114,7 +114,10 @@ class LongerCommodityCodeController @Inject() (
     request.userAnswers
       .get(RecordCategorisationsQuery)
       .flatMap(_.records.get(recordId))
-      .flatMap(_.originalCommodityCode.map(_.reverse.dropWhile(_ == "0").reverse.padTo(validLength, "0").mkString))
+      .flatMap(
+        _.originalCommodityCode
+          .map(_.reverse.dropWhile(char => char == '0' || char == "0").reverse.padTo(validLength, "0").mkString)
```

Comment:
char == '0' || char == "0" 🫠

Reply:
you got magnifying eyes @ChrisCookOC

Reply:
Don't ask. For some reason, sometimes it would interpret it as a char, or as a string.... it's just a precaution having both. I think it's a local issue but this works.